src/structures/ClientUser.js

```diff
@@ -172,8 +177,3 @@ class ClientUser extends User {
 }
 
 module.exports = ClientUser;
-
-/**
- * @external APIModifyClientData
- * @see {@link https://discord.com/developers/docs/resources/user#modify-current-user-json-params}
```

Comment:
Any reason to remove this?

Reply:
Yeah I added a typedef above for this, it should've never been added in the first place I just read it wrong when I did it